### PR TITLE
support for two important features of keras: multiple inputs and customizable metrics

### PR DESCRIPTION
**Pull request by CY-dev**
- modified the "prefetching" method of Worker to allow features_column to be a list of strings representing multiple inputs that are already supported in keras (very important for building complex models)
- modified comments for "features_col" argument in "Trainer"s correspondingly 
- added "metrics" argument to "Trainer" and its subclasses so users can supply customized metrics other than "accuracy"

### CODE REVIEW

**Comment by JoeriHermans** [comment]
Could you provide me with a Keras model where this would be useful? Because personally I would flatten everything.

Joeri

**Comment by CY-dev** [comment]
Here is a good example of retweet prediction on Keras documentation:
https://keras.io/getting-started/functional-api-guide/#multi-input-and-multi-output-models

On our side, we’re trying out a somewhat similar model on hotel sorting. In this case we have search features, hotel features etc. as inputs instead of tweets.

Thanks,
Congrui

File: README.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ In order to submit a job to a remote cluster, you simply run the following code:
 
 ```python
 # Define the distributed optimization procedure, and its parameters.
-trainer = ADAG(keras_model=mlp, worker_optimizer=optimizer_mlp, loss=loss_mlp, num_workers=20,
+trainer = ADAG(keras_model=mlp, worker_optimizer=optimizer_mlp, loss=loss_mlp, metrics = ["accuracy"], num_workers=20,
```
Comment (JoeriHermans) [inline README.md]: In order to preserve consistency, could you add "accuracy" as the default metric for all optimizers?

File: distkeras/trainers.py [hunk 2]
```diff
@@ -46,13 +46,16 @@ class Trainer(object):
               See: https://keras.io/objectives/
         worker_optimizer: string. String representing worker optimizer.
                           See https://keras.io/optimizers/
+        metrics: list of strings representing model evaluation metrics.
+                 See: https://keras.io/objectives/
     """
 
-    def __init__(self, keras_model, loss, worker_optimizer):
+    def __init__(self, keras_model, loss, worker_optimizer, metrics=["accuracy"]):
```
Comment (JoeriHermans) [inline distkeras/trainers.py]: Same as in workers.py, to preserve consistency, could you add "accuracy" as the default metric for all trainers?

You added it in the docs, but not in the code :)

Comment (CY-dev) [inline distkeras/trainers.py]: Requested changes are made. Look ok?

**Comment by JoeriHermans** [comment]
Looks good. Thanks again.
